--- /usr/include/FreeImage.h	2016-03-18 00:15:54.929544463 +0100
+++ FreeImage.h	2016-03-18 00:16:45.717542667 +0100
@@ -136,11 +136,11 @@
 #ifndef _MSC_VER
 // define portable types for 32-bit / 64-bit OS
 #include <inttypes.h>
-typedef int32_t BOOL;
+typedef unsigned int BOOL;
 typedef uint8_t BYTE;
 typedef uint16_t WORD;
 typedef uint32_t DWORD;
-typedef int32_t LONG;
+typedef unsigned int LONG;
 #ifndef _LIBRAW_TYPES_H
 typedef int64_t INT64;
 typedef uint64_t UINT64;
